jobs/wazuh-agent/templates/config/ossec_cluster.conf.erb

```diff
@@ -0,0 +1,208 @@
+<!--
+  Wazuh - Agent - Bosh configuration
+  More info at: https://documentation.wazuh.com
+  Mailing list: https://groups.google.com/forum/#!forum/wazuh
+-->
```

Comment:
Can you consider putting a reference to the source version? and url of the template that we use as a reference?

Reply:
After chatting about it, consider dismissing this for now. Since the template is inherited from the source branch,